credentials/alts/internal/handshaker/service/service_test.go

```diff
@@ -53,7 +60,9 @@ func TestDial(t *testing.T) {
 		t.Fatalf("hsConnMap[%v]=%v, want %v", testAddress1, got, want)
 	}
 
-	// Second call to Dial should return conn1 above.
+	// Second call to Dial with the same address returns the existing connection
```

Comment:
nit: these inline comments can be shortened now if you modify the docstring above